fixtures/tracks/jewels/exercises/TRACK_HINTS.md

```diff
@@ -0,0 +1 @@
+This is the content of the track hints file
```

Comment:
Because this creates an `exercises` dir in `jewels`, the `hello-world` exercise in `jewels` will have no files.